ogre/src/OgreCamera.cc

```diff
@@ -66,6 +66,15 @@ void OgreCamera::Destroy()
   }
 }
 
+//////////////////////////////////////////////////
+math::Angle OgreCamera::HFOV() const
+{
+  double vfov = this->ogreCamera->getFOVy().valueRadians();
+  double hFOV;
+  hFOV = 2.0 * atan(tan(vfov / 2.0) * this->ImageWidth() / this->ImageHeight());
```

Comment:
Currently this causes a failure in `UNIT_Camera_TEST`:
```
  /github/workspace/src/Camera_TEST.cc:75: Failure
  Expected equality of these values:
    hfov.Radian()
      Which is: 1.5700000000000001
    camera->HFOV().Radian()
      Which is: 1.2862402200698853
```

I suggest you update this section of code (and for `Ogre2`) as so:

```suggestion
  double hfov = 2.0 * atan(tan(vfov / 2.0) * this->AspectRatio());
```

You will also need to update: https://github.com/gazebosim/gz-rendering/blob/1e415bb1a3d374b15d9c6cb86be76b2b269711fb/src/Camera_TEST.cc#L75

to use `EXPECT_NEAR`

Reply:
fixed in df3d48d.